src/components/Pass/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { h, Component } from 'preact'
+import { trackComponent } from '../../Tracker'
+import ScreenLayout from '../Theme/ScreenLayout'
+import PageTitle from '../PageTitle'
+import { localised } from '../../locales'
+import { WithLocalisedProps, WithTrackingProps } from '~types/hocs'
+import { StepComponentBaseProps } from '~types/routers'
+import theme from '../Theme/style.scss'
+import style from './style.scss'
+
+type Props = {
+  nextStep?: () => void
+} & WithLocalisedProps &
+  WithTrackingProps &
+  StepComponentBaseProps
+
+class Pass extends Component<Props> {
```

Comment:
Do a quick diff between Pass and Reject component. The only differ in the translation key & class name. Please extract to a common codebase.

Reply:
the 2 extra screens are removed now.